llc/llc-evolution.md

```diff
@@ -103,16 +103,16 @@ public virtual Response GetPet(string id, RequestOptions requestOptions = defaul
 public virtual Response CreateOrUpdatePet(string id, RequestContent body, RequestOptions requestOptions = default);
 
 // Hand authored overloads
-public virtual Response<Pet> GetPet(string id, CancellationToken cacellationToken = default);
-public virtual Response CreateOrUpdatePet(string id, Pet pet, CancellationToken cacellationToken = default);
+public virtual Response<Pet> GetPet(string id, CancellationToken cancellationToken = default);
+public virtual Response CreateOrUpdatePet(string id, Pet pet, CancellationToken cancellationToken = default);
 ```
 
-The evolution pattern here is to replace `RequestOptions` with `CancellationToken` in method signature, replace `RequestConent` with a model type that represents the body of the operation and to use `Response<T>` with a model type for operations which return values.
+The evolution pattern here is to replace `RequestOptions` with `CancellationToken` in method signature, replace `RequestContent` with a model type that represents the body of the operation and to use `Response<T>` with a model type for operations which return values.
```

Comment:
Per discussions I've had with @KrzysztofCwalina lately, it sounds like he is thinking that HLC methods will have no CancellationToken parameter, and LLC methods will take RequestOptions, to prevent the issue around methods that don't take a body.  Would it make sense to update this, given that?

Reply:
Yes - I can rewrite this based on that observation.